new_platforms/src/optee_common.c

```diff
@@ -7,17 +7,358 @@
 #include <openenclave/bits/report.h>
 #include <openenclave/bits/result.h>
 
+#include <mbedtls/sha256.h>
+#include <mbedtls/x509_crt.h>
+
+#include <RiotStatus.h>
+#include <RiotEcc.h>
+#include <RiotCrypt.h>
+#include <TcpsId.h>
+
 #ifndef _In_
 #include "sal_unsup.h"
 #endif
 
+static unsigned char tcps_oid[] = {0x67, 0x81, 0x05, 0x05, 0x04, 0x02};
```

Comment:
Shouldn't this OID be defined in RIoT/CyRes? (also, lets not call it tcps - maybe device_claims_oid?)

Reply:
cyres_claims_oid moved to tcpsid.